packages/server/lib/controllers/v1/account/signup.ts

```diff
@@ -111,13 +111,13 @@ export const signup = asyncWrapper<PostSignup>(async (req, res) => {
     }
 
     if (!token && flagHasUsage) {
-        const resCreate = await billing.upsertCustomer(account, user);
-        if (resCreate.isErr()) {
-            report(resCreate.error);
+        const linkOrbCustomerRes = await linkOrbCustomer(account, user);
```

Comment:
one of the goal of the `billing` package is to encapsulate all the billing logic so orb stuff don't leak into the app. Can we keep it this way, renaming `utils/orb.ts` to `utils/billing.ts`. Ideally using a generic enough interface (ie: `linkBillingCustomer`( even though it might not be easy to fully abstract orb in the implementation.